### PR TITLE
Add sites to build in github actions 

### PR DESCRIPTION
**Pull request by Rub21**
I have started some actions to build sites in this repository and push to s3.

cc. @batpad

### CODE REVIEW

File: .github/workflows/nominatim.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on: push
+jobs:
+  nominatim:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 10
+      - name: Setup git
```
Comment (geohacker) [inline .github/workflows/nominatim.yaml:11]: @Rub21 I dont think we should use git config and git clone this way. We use the official github action for checkout https://github.com/actions/checkout

**Review by geohacker — CHANGES_REQUESTED** [review]

File: .github/workflows/nominatim.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        if: github.ref == 'refs/heads/main'
+        uses: allenevans/set-env@v2.0.0
+        with:
+          NOMINATIM_API: 'https://nominatim.openhistoricalmap.org/'
+          NOMINATIM_BUCKET: 's3://nominatim.openhistoricalmap.org/'
+          CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.PRODUCTION_CLOUDFRONT_DISTRIBUTION_ID }}
+      - name: Clone and build nominatim-ui
+        if: github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/main'
+        run: |
+          git clone https://github.com/OpenHistoricalMap/nominatim-ui.git
```
Comment (geohacker) [inline .github/workflows/nominatim.yaml:30]: Again, we shouldn't do `git clone` just rely on an existing action module.

**Review by geohacker — CHANGES_REQUESTED** [review]

**Comment by batpad** [comment]
Yup - @Rub21 this looks great to me - would just be great to switch to using the github action module and clearly indicate the repo and git tag / commit to use from that repo. And then I think we could make similar actions for all the frontend repositories as per https://github.com/OpenHistoricalMap/issues/issues/477

**Comment by Rub21** [comment]
@batpad @geohacker I am going to merge this PR to staging , in order to see that is working fine, there, locally it is working good!

**Pull request merged**